cmd/acraserver/acraserver.go

```diff
@@ -152,7 +158,13 @@ func main() {
 		}
 	}
 
-	server, err := NewServer(config, keyStore)
+	var server *SServer
+	if os.Getenv("_GRACEFUL_RESTART") == "true" {
```

Comment:
What about to pass value from env variable through [string.ToLower](https://golang.org/pkg/strings/#ToLower) to support values `True`|`TRUE`?

Reply:
I would move this `if` to a separate function, like 

```
func isGracefulRestart() bool {
	return strings.ToLower(os.Getenv("_GRACEFUL_RESTART")) == "true"
}
```

because we are checking it's value in many places, better to declare it once (in case we will rename env variable)

Reply:
maybe more `"_GRACEFUL_RESTART"` as constant as well?

```
var gracefulRestartSetting string = "_GRACEFUL_RESTART"
```